README.md

```diff
@@ -69,6 +69,7 @@ NOTE:
 |        | Provision volume from another volume                      | -              | -                  | -                | -                    | -                  |
 |        | Resize volume                                             | Beta           | >= v2.0.0          | >= v1.1.0        | Mimic (>=v13.0.0)    | >= v1.15.0         |
 |        | Metrics Support                                           | Beta           | >= v1.2.0          | >= v1.1.0        | Mimic (>=v13.0.0)    | >= v1.15.0         |
+|        | Topology Aware Provisioning Support                       | Alpha          | >= v2.1.0          | >= v1.2.0        | Mimic (>=v13.0.0)    | >= v1.17.0         |
```

Comment:
@humblec can you check minimum supported kube version https://kubernetes-csi.github.io/docs/topology.html

Reply:
@Madhu-1  it is GAd in kube 1.17 GA..  so the doc update.

Reply:
Miminum version can be beta isn't it?

Reply:
It can be , however, based on some of the fixes in this area and may be by its reference to CSI spec ..etc its better to keep v1.17, so that no hiccups.

Reply:
IMO it's better to keep minimum version to beta as we did for metrics and resize. 
From the doc, kube 1.17 need external-provisioner 1.5 which is not true now, in cephcsi we are using 1.4

Reply:
ok.. Done.